support/k8s/k8s-workload-registrar/config_reconcile.go

```diff
@@ -69,6 +69,8 @@ func (c *ReconcileMode) Run(ctx context.Context) error {
 	}))
 	setupLog := ctrl.Log.WithName("setup")
 
+	setupLog.Info("The k8s-workload-registrar is deprecated and no longer maintained. Please migrate to the SPIRE Controller Manager (https://github.com/spiffe/spire-controller-manager).")
```

Comment:
should be at Warning level instead of Info?

Reply:
`logr` loggers only have info and error levels. I considered logging at error level but in the end determined this was more info than error :)

Reply:
Fair enough.